seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/client/ServiceProperties.java

```diff
@@ -85,12 +91,37 @@ public ServiceProperties setDisableGlobalTransaction(boolean disableGlobalTransa
         return this;
     }
 
+    @Autowired(required = false)
```

Comment:
Why not annotate it directly to the field？

Reply:
for the unit test

Reply:
The set method is added to test cases. `@Autowired(required = false)` can also annotate it to the field.

Reply:
I annotated it to the properties.